src/routers/categories.js

```diff
@@ -0,0 +1,42 @@
+const express  = require('express')
+const Category = require('../models/category')
+const router = express.Router()
+
+const TYPES = {
```

Comment:
moze stworzyc jakis plik z const'ami i wrzucic to tam wtedy bedziesz mogl uzyc tych typew w enumie w modelu

Reply:
poprawione